docs/prow.md

```diff
@@ -0,0 +1,397 @@
+
+When a periodic E2E job runs as part of an EV2 pipeline, you can identify the associated EV2 rollout by examining the job's metadata:
+
+1. **View Job Details**: Click on a specific job run in the [Prow dashboard](https://prow.ci.openshift.org/?repo=Azure%2FARO-HCP)
+
+2. **Check Annotations**: Look for the `ev2.rollout/` prefix in the job annotations, which provide:
+   - **Rollout ID**: Unique identifier for the EV2 rollout (e.g., `ev2.rollout/ARO-HCP: "665a88398919"`)
+   - **Build Number**: The ADO build number (e.g., `ev2.rollout/build: "144984048"`)
+   - **Region**: The Azure region being tested (e.g., `ev2.rollout/region: "uksouth"`)
+   - **SDP Pipeline**: The SDP pipeline identifier (e.g., `ev2.rollout/sdp-pipelines: "a848ce2e"`)
+
+3. **Example Job URL**:
+
+   ```text
+   https://prow.ci.openshift.org/prowjob?prowjob=b2054fce-2218-4d65-8b20-2bc4a3a9df51
+   ```
+
+   This job will display the associated EV2 rollout information in its metadata.
+
+### Promotion Gating
+
+For environments where `gatePromotion: true` is set (like Integration), the success of the Prow E2E tests can gate the promotion to the next environment. This ensures that only validated deployments proceed through the release pipeline.
+
+### For More Information
+
+- See the [E2E Testing documentation](../test/e2e/README.md) for details on running and writing E2E tests
+- See the [Pipelines documentation](pipelines.md) for information about the deployment pipeline system
+- See the [EV2 Deployment documentation](ev2-deployment.md) for details on EV2 deployment processes
+
+## Working with Prow Jobs
+
+### Triggering Presubmit Jobs
+
+To trigger optional presubmit jobs on a pull request, add a comment to the PR with the appropriate test command:
+
+```text
+/test e2e-parallel
+```
+
+To re-run a failed job:
+
+```text
+/retest
+```
+
+To re-run a specific job:
+
+```text
+/test <job-trigger>
+```
+
+### Viewing Job Results
+
+Job results are reported directly on pull requests as GitHub checks. You can:
+
+1. View the status of all jobs in the PR's "Checks" tab
+2. Click on a specific job to see detailed logs and test output
+3. Access the Prow dashboard for more detailed information and job history
+
+### Job Execution Environment
+
+All Prow jobs run on OpenShift clusters managed by the OpenShift CI infrastructure:
+
+- Jobs use Kubernetes for orchestration
+- Jobs run with ci-operator for standardized build and test workflows
+- Most jobs run on the build02 cluster, with some specialized jobs on build07 or build10
+
+### Modifying Prow Jobs
+
+Prow job definitions are maintained in the [openshift/release](https://github.com/openshift/release) repository, not in this repository. The actual job files are **generated** from configuration files, so you must edit the configs and regenerate.
+
+To modify or add jobs:
+
+1. Fork the [openshift/release](https://github.com/openshift/release) repository
+2. Edit configuration files in `ci-operator/config/Azure/ARO-HCP/`:
```

Comment:
```suggestion
2. Edit configuration files in [`ci-operator/config/Azure/ARO-HCP/`](https://github.com/openshift/release/tree/master/ci-operator/config/Azure/ARO-HCP):
```

Reply:
Fixed in https://github.com/Azure/ARO-HCP/pull/3424/commits/71481d15768e635c9d17f3e4c3fe8eda3505b94f